polaris.shopify.com/src/components/FoundationsPage/FoundationsPage.tsx

```diff
@@ -45,7 +45,7 @@ function FoundationsPage({
           <p>{description}</p>
         </Longform>
         <Stack gap="8">
-          {typedStatus ? <StatusBanner status={typedStatus} /> : null}
+          {typedStatus && <StatusBanner status={typedStatus} />}
```

Comment:
This can result in rendering erroneous `false` values if the left-hand side of the binary expression (`&&`) returns `false` - react will happily render `false` as a string in the final output.

The ternary syntax (`<condition> ? <if true> : <if false>`) allows us to control what the expression returns, ie; `null`, which React will not render.

```suggestion
          {typedStatus ? <StatusBanner status={typedStatus} /> : null}
```